bouter/free/__init__.py

```diff
@@ -136,12 +136,13 @@ def compute_velocity(
         return fish_velocities
 
     @decorators.cache_results()
-    def get_bouts(self, scale=None, threshold=1, **kwargs):
+    def get_bouts(self, scale=None, threshold=1, conv_detection=False, **kwargs):
         """Extracts all bouts from a freely-swimming tracking experiment
 
         :param exp: the experiment object
         :param scale: mm per pixel, recalculated by default
-        :param threshold: velocity threshold in mm/s
+        :param threshold: velocity threshold in mm/s or score threshold if conv_detection=True
```

Comment:
I wonder if the fact that this threshold is provided via the same argument can lead to small issues with people not realizing the thresholds for both methods are in different units, and therefore not detecting bouts properly.
One could split this into two different threshold arguments, but maybe it makes the function uglier with lots of arguments. Alternatively, maybe adding a warning if your method is used, pointing towards thinking about the provided threshold?